decidim-elections/app/cells/decidim/votings/voting_m_cell.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # This cell renders the Medium (:m) question card
+    # for an given instance of a Question
+    class VotingMCell < Decidim::CardMCell
+      private
+
+      def has_state?
+        false
+      end
+
+      # Even though we need to render the badge, we can't do it in the normal
+      # way, because the paragraph comes from a user input and contains HTML.
+      # This causes the badge and the paragraph to appear in different lines.
+      # In order to fix it, check the `description` method.
+      def has_badge?
+        false
+      end
+
+      def badge_name
+        text = model.period_status
+        return unless text
+
+        I18n.t(text, scope: "decidim.votings.votings_m.badge_name")
+      end
+
+      # In order to render the badge inline with the paragraph text we need to
+      # find the opening `<p>` tag and include the badge right after it. This
+      # makes the layout look good.
+      def description
+        text = super
+        text.gsub!(/^<p>/, "<p>#{render :badge}")
+        html_truncate(text, length: 100)
+      end
+
+      def resource_path
+        Decidim::Votings::Engine.routes.url_helpers.voting_path(model)
+      end
+
+      def resource_image_path
+        model.banner_image.url
+      end
+
+      def has_image?
+        model.banner_image.url.present?
+      end
+
+      def start_time
+        model.start_time
+      end
+
+      def end_time
+        model.end_time
+      end
+
+      def footer_button_text
+        "view"
```

Comment:
This is actually a reference of an i18n key used [here](https://github.com/decidim/decidim/pull/7160/files#diff-74572f002a7188a69f7888df6ffc42e7356b29d70834c355ad8c22e92978ac99R3).
It is hardcoded to `view` at the moment, but it will probably change to something more dynamic in the near future to change the button's text according to the state of the voting.